src/UI/Components/Diagram/components/FormModal.tsx

```diff
@@ -225,21 +235,19 @@ const FormModal = ({
       >
         <FlexItem>
           <Select
-            selections={selected?.name}
-            onToggle={() => setIsSelectOpen(!isSelectOpen)}
+            isScrollable
+            selected={selected?.name}
+            toggle={toggle}
+            onOpenChange={(isOpen) => setIsSelectOpen(isOpen)}
             isOpen={isSelectOpen}
-            onSelect={(evt, value, isPlaceholder) => {
-              onEntityChosen(evt, value, isPlaceholder, possibleForms);
+            onSelect={(_evt, value) => {
+              onEntityChosen(value, possibleForms);
             }}
-            maxHeight={300}
-            isDisabled={cellView !== null}
           >
             {possibleForms.map(({ key, value }) => (
-              <SelectOption
-                key={key}
-                value={value}
-                isPlaceholder={value === "Choose a Service"}
-              />
+              <SelectOption key={key} value={value}>
+                {value || "Choose a Service"}
```

Comment:
I think having a cross icon in the select toggle to clear the value is better than having an option displaying as Choose a Service